manifests/m/Microsoft/DevHome/0.503.261.0/Microsoft.DevHome.installer.yaml

```
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.DevHome
+PackageVersion: 0.503.261.0

```

```suggestion
PackageVersion: 0.502.261.0
```
The folder will need to be renamed after changing this value